### PR TITLE
[GeckoView] Show the download button by default and add a pref to disable it (bug 1827963)

### PR DESCRIPTION
**Pull request by calixteman**
For the moment there is no real consensus on how we should download a pdf on Android. Hence we keep this solution for the moment but behind a pref (which will be true on nightly only).

### CODE REVIEW

File: web/toolbar-geckoview.js [hunk 1]
```diff
@@ -20,16 +20,24 @@
  * @property {HTMLButtonElement} download - Button to download the document.
  */
 
+const TIME_BEFORE_SHOWING_THE_BUTTON = 200;
```
Comment (Snuffleupagus) [inline web/toolbar-geckoview.js:23]: Given that the toolbar could conceivably, at some later point, contain more than just one button: 
```suggestion
const TIME_BEFORE_SHOWING_TOOLBAR = 200;
```

File: web/app_options.js [hunk 2]
```diff
@@ -113,6 +113,11 @@ const defaultOptions = {
     value: 0,
     kind: OptionKind.VIEWER + OptionKind.PREFERENCE,
   },
+  enablefloatingActionButton: {
```
[the remark this answers is not in the document]
Comment (Snuffleupagus) [reply web/app_options.js:116]: Also, please keep the options sorted alphabetically; hence this should be placed *just after* `disablePageLabels` above.

**Review by Amejia481 — APPROVED** [review]

**Review by Snuffleupagus — APPROVED** [review]
r=me, with a few naming tweaks; thank you!

**Pull request merged**